<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>task</class>
 <widget class="QDialog" name="task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <widget class="XLabel" name="_numberLit">
     <property name="text">
      <string>Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="XLineEdit" name="_number"/>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>46</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="XLabel" name="_statusLit">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XComboBox" name="_status">
         <item>
          <property name="text">
           <string>Concept</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In-Process</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Complete</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="XLabel" name="_nameLit">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="XLineEdit" name="_name"/>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="XLabel" name="_descripLit">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="XTextEdit" name="_descrip">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTabWidget" name="_tab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="_scheduleTab">
      <attribute name="title">
       <string>Schedule</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="_userGroup">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="UsernameCluster" name="_owner">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="label">
             <string>Owner:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UsernameCluster" name="_assignedTo">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="label">
             <string>Assigned to:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="_dateGroup">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="_dueLit">
            <property name="text">
             <string>Due:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="DLineEdit" name="_due">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="_startedLit">
            <property name="text">
             <string>Started:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="DLineEdit" name="_started"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="_assignedLit">
            <property name="text">
             <string>Assigned:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="DLineEdit" name="_assigned"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="_completedLit">
            <property name="text">
             <string>Completed:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="DLineEdit" name="_completed"/>
          </item>
          <item row="2" column="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_planTab">
      <attribute name="title">
       <string>Plan</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="_hoursGroup">
         <property name="title">
          <string>Hours</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="XLabel" name="_budgetHoursLit">
            <property name="text">
             <string>Budgeted:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="XLineEdit" name="_budgetHours">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32767</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="XLabel" name="_actualHoursLit">
            <property name="text">
             <string>Actual:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="XLineEdit" name="_actualHours">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32767</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="XLabel" name="_balanceHoursLit">
            <property name="text">
             <string>Balance:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="XLabel" name="_balanceHours">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="_expensesGroup">
         <property name="title">
          <string>Expenses</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="XLabel" name="_budgetExpLit">
            <property name="text">
             <string>Budgeted:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="XLineEdit" name="_budgetExp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32767</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="XLabel" name="_actualExpLit">
            <property name="text">
             <string>Actual:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="XLineEdit" name="_actualExp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32767</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="XLabel" name="_balanceExpLit">
            <property name="text">
             <string>Balance:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="XLabel" name="_balanceExp">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_alarmTab">
      <attribute name="title">
       <string>Alarms</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="Alarms" name="_alarms">
         <property name="type">
          <enum>Alarms::Project</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_commentsTab">
      <attribute name="title">
       <string>Comments</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="Comments" name="_comments">
         <property name="type">
          <enum>Comments::Task</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_characteristicsTab">
      <attribute name="title">
       <string>Characteristics</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="XTreeWidget" name="_charass">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_newCharacteristic">
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_editCharacteristic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_deleteCharacteristic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DocumentsTab">
      <attribute name="title">
       <string>Documents</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="Documents" name="_documents">
         <property name="type">
          <enum>Documents::ProjectTask</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>Alarms</class>
   <extends>QWidget</extends>
   <header>alarms.h</header>
  </customwidget>
  <customwidget>
   <class>Comments</class>
   <extends>QWidget</extends>
   <header>comments.h</header>
  </customwidget>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Documents</class>
   <extends>QWidget</extends>
   <header>documents.h</header>
  </customwidget>
  <customwidget>
   <class>UsernameCluster</class>
   <extends>QWidget</extends>
   <header>usernamecluster.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_owner</sender>
   <signal>newId(int)</signal>
   <receiver>_alarms</receiver>
   <slot>setUsrId1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_assignedTo</sender>
   <signal>newId(int)</signal>
   <receiver>_alarms</receiver>
   <slot>setUsrId2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_due</sender>
   <signal>newDate(QDate)</signal>
   <receiver>_alarms</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>task</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
